website/docs/reference/global-configs/databricks-changes.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Databricks adapter behavior changes"
+id: "databricks-changes"
+sidebar: "Databricks"
+---
+
+## Behavior Changes
+
+Here are the current [behavior change flags](/docs/reference/global-configs/behavior-changes.md#behavior-change-flags) that are specific to `dbt-databricks`:
+
+| Flag                          | `dbt-databricks`: Intro | `dbt-databricks`: Maturity |
```

Comment:
what do you mean by dbt-databricks intro vs maturity?

Reply:
Let's remove this table and break down by individual flags

Reply:
This is following the table that dbt-core has to summarize behavior flags.

Reply:
@benc-db  If we're going to add this table, can we handle it the same as we do the [Core behavior change table](https://docs.getdbt.com/reference/global-configs/behavior-changes#behavior-change-flags) and include the dbt Cloud intro and permanent dates?